vtex-apps/responsive-layout
# First implementation of responsive-layout blocks

#### What does this PR do?

Creates four new blocks:

- `responsive-layout.desktop`
- `responsive-layout.mobile`
- `responsive-layout.tablet`
- `responsive-layout.phone`

Each of those blocks has composition `children`, and will only render their children in the correct screen-sizes.

#### How to test it? \*

Just resize the window and see the magic take place:
https://victorhmp--storecomponents.myvtex.com/about-us
